tagilmo/utils/malmo_wrapper.py

```diff
@@ -309,6 +309,8 @@ def __init__(self, mc, nAgent = 0):
             self.canBeNone.append('getImageFrame')
         if not self.mc.supportsSegmentation():
             self.canBeNone.append('getSegmentationFrame')
+        self.max_dt = {method: 1.0 for method in self.methods}
+        self.max_dt['getChat'] = 0.1
```

Comment:
isn't it too small? We can miss event when chat is not None

Reply:
> isn't it too small? We can miss event when chat is not None

if we're using 1.0 as dt for get_chat agent start to act inadequate. Moreover, it is only for get_chat method, others uses 1.0